modules/ROOT/pages/storage.adoc

```diff
@@ -312,7 +312,7 @@ The partition table layout has changed over time. The current layout is:
 
 The EFI-SYSTEM partition can be deleted or reformatted when BIOS booting. Similarly, the BIOS-BOOT partition can be deleted or reformatted when EFI booting.
 
-== Mounted Filesystems
+== Filesystem Configuration
```

Comment:
Hmm.  This will break any existing links to the section, which we may or may not care about.  But also, "Configuration" makes it sound like the section provides guidance on configuring filesystems, when in fact it describes the design and default layout of the filesystem hierarchy.

Reply:
How about just "Filesystems" ?

Reply:
I can live with it, but it seems pretty generic.  I don't have a better suggestion at the moment, though.  It also seems okay to stick with the original heading for now.

Reply:
OK, in the interest of avoiding churn and breaking links, I just left the name the same.